iib/web/iib_static_types.py

```diff
@@ -390,7 +390,7 @@ class MergeIndexImageRequestResponse(APIPartImageBuildRequestResponse):
     build_tags: List[str]
     deprecation_list: List[str]
     distribution_scope: str
-    graph_update_mode: Literal['replaces', 'semver', 'semver-skippatch']
+    graph_update_mode: GRAPH_MODE_LITERAL
```

Comment:
Is it possible to do this in the previous commit instead of this one?

Reply:
The other way around. The graph_update_mode should not be present in the first commit.